tests/testthat/test-cols_align.R

```diff
@@ -1,13 +1,12 @@
-context("Ensuring that the `cols_align*()` functions work as expected")
```

Comment:
Just flagging that I see a number of these `context()` calls being removed and not added back anywhere.

Reply:
The `context()` function is deprecated in `testthat` 3rd edition. For that reason, all are being removed in this migration.